jinahub/indexers/searcher/FaissSearcher/tests/requirements.txt

```diff
@@ -1,4 +1 @@
-jina==2.0.21
-faiss-cpu==1.6.5
```

Comment:
Just out of interest: Why don't we need this anymore?

Reply:
They are installed by the main `requirements.txt`.